Configs/App-Shared.xcconfig

```diff
@@ -7,4 +7,7 @@
 ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon
 
 // The compiler to use for C, C++, and Objective-C.
-GCC_VERSION =
\ No newline at end of file
+GCC_VERSION =
```

Comment:
What's the purpose of not setting any value?

Reply:
I don't know. This is the xcconfig writing by @bpoplauschi . And this applied for only the `SDWebImage Example` project, does not effect the framework itself.

Reply:
Seems a deprecated build settings from the day of Xcode 5...Since now all Xcode using the clang compiler, I remove this line and still can run the demo.